synapse/event_auth.py

```diff
@@ -688,7 +688,7 @@ def _verify_third_party_invite(event: EventBase, auth_events: StateMap[EventBase
     return False
 
 
-def get_public_keys(invite_event):
+def get_public_keys(invite_event: EventBase) -> List[Dict[str, str]]:
```

Comment:
I'm not 100% confident that the values here are strings, but I think they are?

Reply:
I think so too, but not 100% sure. I wonder if we want to be saying they are strings if we haven't actually asserted that? The event content is untrusted data after all

Reply:
Would `Dict[str, Any]` make more sense?

Reply:
Yeah, I'd vote we either assert they're strings or say they're `Any`?

Reply:
Done!